lib/teleterm/api/uri/urlpath.go

```diff
@@ -0,0 +1,241 @@
+/*MIT License
+
+Copyright (c) 2019 Ulysse Carion
```

Comment:
I diffed it against [the original source](https://github.com/ucarion/urlpath) and it looks like we didn't change anything in this package. I don't know why we're vendoring it.

Since it seems that we don't need to vendor it, do you recommend we import it as any other Go module?